grazer/src/main/webapp/WEB-INF/views/fileUploadForm.jsp

```diff
@@ -0,0 +1,53 @@
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<div id="global">
+    <form:form modelAttribute="transformation" commandName="transformation" action="/grazer/transformation/save?${_csrf.parameterName}=${_csrf.token}" method="post" enctype="multipart/form-data">
+        <fieldset>
+            <legend>Add a file</legend>
+            <form:errors path = "*" cssClass = "errorblock" element = "div" style="color:red"/>
```

Comment:
would this show all the errors and therefore duplicate the errors below?

Reply:
Yes this shows all the errors at the top and lists them below each corresponding field as well. Should I remove this line?